cirq/ops/pauli_string.py

```diff
@@ -171,7 +171,7 @@ def __getitem__(self, key: 'cirq.Qid') -> pauli_gates.Pauli:
 
     # pylint: disable=function-redefined
     @overload
-    def get(self, key: 'cirq.Qid') -> pauli_gates.Pauli:
+    def get(self, key: 'cirq.Qid', default: None = None) -> pauli_gates.Pauli:
```

Comment:
Can you explain this change? I assume it's for the type checking? How is it different than the final definition
```python
def get(self, key: 'cirq.Qid', default=None):
```

Reply:
Type-checking succeeds either way, but without explicit specification of the `default` parameter in one of the overloads pylint complains when we try to use it.